### PR TITLE
Update preamble.tex with XeLaTeX packages

### PR DESCRIPTION
**Pull request by hodie**
Added some German-specific packages for XeLaTex

### CODE REVIEW

**Comment by jdhoffa** [comment]
I should have mentioned, if we are adding any new Tex dependencies, those must also be added to the workflow Dockerfile:

https://github.com/RMI-PACTA/workflow.transition.monitor/blob/main/Dockerfile#L49-L74

**Comment by hodie** [comment]
> I should have mentioned, if we are adding any new Tex dependencies, those deps must also be added to the workflow Dockerfile, otherwise they won't be available on build:
> 
> https://github.com/RMI-PACTA/workflow.transition.monitor/blob/main/Dockerfile#L49-L74

Ah @jdhoffa  that makes sense why the German Babel package gave me an error. Or do you mean just for the real build, not the CI test builds?

**Comment by hodie** [comment]
Yeah, sorry, I tried adding that babel package over the weekend. It should have added proper hyphenation but despite adding it as a dependency it just doesn't seem to work. We might just have to comment that out and live without hyphenation.


[photo-logo]<https://img.newoldstamp.com/r/684790/p>
hodie Meyers
he/him
Technical Product Director RMI
Climate Finance
m: +1(303) 335-0330
e: ***@***.***<https://img.newoldstamp.com/r/684790/e?id=1>
[facebook]<https://img.newoldstamp.com/r/684790/facebook>
[twitter]<https://img.newoldstamp.com/r/684790/twitter>
[linkedin]<https://img.newoldstamp.com/r/684790/linkedin>
[instagram]<https://img.newoldstamp.com/r/684790/instagram>
[mail]<https://img.newoldstamp.com/r/684790/mail>
[banner]<https://img.newoldstamp.com/r/684790/b>

***@***.***?anonymous&ep=signature>          Book time to meet with ***@***.***?anonymous&ep=signature>
________________________________

**Comment by jdhoffa** [comment]
Depends on https://github.com/RMI-PACTA/workflow.transition.monitor/pull/378

I will wait until after the new Tex deps are added to WTM to merge this PR, to ensure we never have a breaking CI build.

File: inst/extdata/PA2024CH_de_exec_summary/preamble.tex
Comment (hodie) [reply inst/extdata/PA2024CH_de_exec_summary/preamble.tex:19]: I think rows 17-19 should got, not just 19?

File: inst/extdata/PA2024CH_de_exec_summary/preamble.tex
Comment (jdhoffa) [inline inst/extdata/PA2024CH_de_exec_summary/preamble.tex:17]:
```suggestion
```

**Review by jdhoffa — APPROVED** [review]

**Pull request merged**
